### PR TITLE
Resolve dictionary test

### PR DESCRIPTION
**Pull request by salcudeann**
To run this problem, i had to change _be_true_ and _be_false_ from dictionary_spec.rb in _be_truthy_ and _be_falsey_.

### CODE REVIEW

File: 11_dictionary/dictionary.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    else
+      key.each do |k, v|
+        @hash[k] = v
+      end
+    end
+  end
+
+  def keywords()
+    key_array = []
+    @hash.each { |k, v| key_array.push(k) }
```
Comment (pirvudoru) [inline 11_dictionary/dictionary.rb]: Ahem... https://ruby-doc.org/core-3.0.1/Hash.html#method-i-keys

File: 11_dictionary/dictionary.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    key_array.sort
+  end
+
+  def include?(key)
+    @hash.has_key?(key) ? true : false
```
Comment (pirvudoru) [inline 11_dictionary/dictionary.rb]: What are the possible values that `has_key?` can return? :wink:

File: 11_dictionary/dictionary.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    @hash
+  end
+
+  def add(key)
+    if key.is_a? String
+      @hash[key] = nil
+    else
+      key.each do |k, v|
+        @hash[k] = v
+      end
```
Comment (alex-damian-negru) [inline 11_dictionary/dictionary.rb]: https://apidock.com/ruby/v2_6_3/Hash/merge check this out.

File: 11_dictionary/dictionary.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    end
+  end
+
+  def keywords()
```
Comment (alex-damian-negru) [inline 11_dictionary/dictionary.rb]: The `()` are not required in ruby if there are no params.

File: 11_dictionary/dictionary.rb [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    key_array = []
+    @hash.each { |k, v| key_array.push(k) }
+    key_array.sort
```
Comment (alex-damian-negru) [inline 11_dictionary/dictionary.rb]: There's a method in Ruby that allows you to fetch all the keys for a Hash, try to use that.

File: 11_dictionary/dictionary.rb (hunk 2 above)
Comment (alex-damian-negru) [inline 11_dictionary/dictionary.rb]: `has_key?(key)` already returns a bool.

**Review by alex-damian-negru — CHANGES_REQUESTED** [review]

**Pull request merged**